src/styles/main.scss

```diff
@@ -0,0 +1,3 @@
+@import "./utils/mixins";
+@import "./blocks/page.scss";
```

Comment:
The file extension '.scss' should be removed from the import statement. Use '@import "./blocks/page";' instead.